tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -758,6 +759,98 @@ public void OdbcShorthandNotationTest()
             Assert.Equal(typeof(System.Data.Odbc.OdbcConnection), dt.ConnectionType);
         }
 
+        [Fact]
+        public void Oracle_SetParamTypeShouldSetCustomEnumValueTest()
```

Comment:
Can you clone this test case and edit it for SQL server? Use [SqlParameter](https://msdn.microsoft.com/en-us/library/system.data.sqlclient.sqlparameter(v=vs.110).aspx)